### PR TITLE
Spice 0.18.1-beta release notes

### PR DESCRIPTION
**Pull request by sgrebnov**
## 🗣 Description

Spice 0.18.1-beta release notes

### CODE REVIEW

File: docs/release_notes/v0.18.1-beta.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    acceleration:
+      enabled: true
+      refresh_mode: full
+      params:
+        file_watcher: enabled
+```
+
+## Breaking Changes
+
+`runtime.query_history` was depricated and not available. Use `runtime.task_history`.
```
Comment (ewgenius) [inline docs/release_notes/v0.18.1-beta.md]:
```suggestion
`runtime.query_history` has been deprecated and is no longer available. Please use `runtime.task_history` instead.
```

File: docs/release_notes/v0.18.1-beta.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    task_history:
+        enabled: true
+        captured_output: truncated
+        retention_period: 8h
+        retention_check_interval: 15m
+```
+
+See the [Task History Spicepod reference](https://docs.spiceai.org/reference/spicepod#runtimetask_history) for more information on possible values and behaviors.
+
+**spice search** Use `spice search` CLI command to perform embeddings-based searches across connected datasets
```
Comment (ewgenius) [inline docs/release_notes/v0.18.1-beta.md]:
```suggestion
**spice search** Use `spice search` CLI command to perform embeddings-based searches across connected datasets.
```

File: docs/release_notes/v0.18.1-beta.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# Spice v0.18.1-beta (Sep 23, 2024)
+
+The v0.18.1-beta release enhances performance for running accelerated queries across multiple datasets, introduces configurable Task History behavior, adds support for the `mssql` data connector, and improves the reliability and error handling for the `graphql`, `mysql` and `sharepoint` data connectors. It also introduces a new `spice search` CLI command, allowing users to perform embeddings-based searches across connected datasets, and adds automatic refresh triggers for the `file` data connector when changes are detected in the source data.
```
Comment (Jeadie) [inline docs/release_notes/v0.18.1-beta.md]: This should be > 1 sentence since we have so much in this release.

Comment (Jeadie) [inline docs/release_notes/v0.18.1-beta.md]:
```suggestion
The v0.18.1-beta release enhances performance for running accelerated queries across multiple datasets, introduces configurable Task History behavior, adds support for the `mssql` data connector, and improves the reliability and error handling for the `graphql`, `mysql` and `sharepoint` data connectors. It also introduces a new `spice search` CLI command, allowing users to perform vector searches across connected datasets, and adds automatic refresh triggers for the `file` data connector when changes are detected in the source data.
```

Comment (eadgbear) [inline docs/release_notes/v0.18.1-beta.md]: Did someone open a task to document `spice search`?

**Review by phillipleblanc — DISMISSED** [review]

File: docs/release_notes/v0.18.1-beta.md (hunk 3 above)
Comment (sgrebnov) [reply docs/release_notes/v0.18.1-beta.md]: I'll add a page while we are building spice

**Review by eadgbear — APPROVED** [review]

**Pull request merged**